pkg/generate/code/set_sdk.go

```diff
@@ -31,44 +31,46 @@ import (
 // Assume a CRD called Repository that looks like this pseudo-schema:
 //
 // .Status
-//   .Authors ([]*string)
-//   .ImageData
-//     .Location (*string)
-//     .Tag (*string)
-//   .Name (*string)
+//
+//	.Authors ([]*string)
+//	.ImageData
+//	  .Location (*string)
+//	  .Tag (*string)
+//	.Name (*string)
```

Comment:
Weird from my new gofmt (using go1.20)

Reply:
yeah, no worries. we should probably do a single style-only commit that updates gofmt for this repository (and runtime)

Reply:
> yeah, no worries. we should probably do a single style-only commit that updates gofmt for this repository (and runtime)

@A-Hilaly et voila:
* https://github.com/aws-controllers-k8s/runtime/pull/110
* https://github.com/aws-controllers-k8s/code-generator/pull/407